src/styles/getTooltipStyles.js

```diff
@@ -0,0 +1,117 @@
+import {Dimensions} from 'react-native';
+import spacing from './utilities/spacing';
+import styles from './styles';
+import themeColors from './themes/default';
+import fontFamily from './fontFamily';
+import variables from './variables';
+
+const GUTTER_WIDTH = 16;
+
+/**
+ * The Expensify.cash repo is very consistent about doing spacing in multiples of 4.
+ * In an effort to maintain that consistency, we'll make sure that any distance we're shifting the tooltip
+ * is a multiple of 4.
+ *
+ * @param {Number} n
+ * @returns {Number}
+ */
+function roundToNearestMultipleOfFour(n) {
+    if (n > 0) {
+        return Math.ceil(n / 4.0) * 4;
+    }
+
+    if (n < 0) {
+        return Math.floor(n / 4.0) * 4;
+    }
+
+    return 0;
+}
+
+export default function getTooltipStyles(
+    interpolatedSize,
+    xOffset,
+    yOffset,
+    width,
+    height,
```

Comment:
`width` and `height` should have more descriptive names? It's not clear what they relate to.